app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/filesdetails/NewTransferFilesDetailsScreen.kt

```diff
@@ -79,7 +79,7 @@ private fun NewTransferFilesDetailsScreen(
     navigateBack: (() -> Unit),
 ) {
     val snackbarHostState = remember { SnackbarHostState() }
-    Scaffold(
+    SinglePaneScaffold(
```

Comment:
It was quicker to fix than I thought. Is this the only screen that's missing this horizontal maximum width constraint?